src/dlgConnectionProfiles.cpp

```diff
@@ -52,7 +52,8 @@ dlgConnectionProfiles::dlgConnectionProfiles(QWidget * parent)
     profiles_tree_widget->setSelectionMode(QAbstractItemView::SingleSelection);
 
     QAbstractButton* abort = dialog_buttonbox->button(QDialogButtonBox::Cancel);
-    connect_button = dialog_buttonbox->addButton(tr("Connect"), QDialogButtonBox::AcceptRole);
+    connect_button = dialog_buttonbox->button(QDialogButtonBox::Apply);
+    connect_button->setText("Co&nnect");
```

Comment:
It looks as though the aim was to change the `Connect` button to have an `QDialogButtonBox::Apply` role instead of an `QDialogButtonBox::Accept` one - IIRC the latter causes the dialogue to close after the button is clicked whereas the former is intended for something to *happen* on the dialogue but to keep it open. It may be that the OP needs this change and spotted that the dialogue button box could be created with the button already in place with no reason to have to add it later as the previous code did (I agree that does seem a reasonable refactor step).

I agree that they should have kept the `tr(`...`)` around the button text - thought I also have reservations about the addition of an `<Alt>-N` short-cut (with the `&`) as we will shortly be getting into the difficult area of providing end-user language dependent short-cuts and adding one here is a little premature.

Reply:
Talking about tab order, I like that very much. However it seems like your new option "Reconnect on disconnect" is not included there? My tab order moves directly from "Auto connect on startup" over to "Character name". Is that intended behavior?